tests/unit/data/ar/Animal.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yiiunit\data\ar;
+
+/**
+ * Class Animal
+ *
+ * @author Jose Lorente <jose.lorente.martin@gmail.com>
+ * @property integer $id
+ * @property string $type
+ */
+class Animal extends ActiveRecord {
+
+    public $does;
+
+    public static function tableName() {
```

Comment:
The curly bracket needs to go to the next line, according to our coding style. Please fix them all in this PR.